tests/test_integration_oidc_flow_variants.py

```diff
@@ -110,3 +113,75 @@ def test_return_jwt_from_userinfo_endpoint(
 
     # a user was created
     assert django_user_model.objects.count() == 1
+
+
+@pytest.mark.vcr
+def test_session_refresh(
+    keycloak_config,
+    settings,
+    mock_state_and_nonce,
+    client,
+    django_user_model,
+    vcr,
+    mocker,
+):
+    session = requests.Session()
+
+    keycloak_config.make_users_staff = True
+    keycloak_config.save()
```

Comment:
you can use `@pytest.mark.oidcconfig(make_users_staff=True)` instead so you can focus on the actual test in the function body